supervisor/misc/tasks.py

```diff
@@ -333,3 +334,12 @@ async def _watchdog_addon_application(self):
     async def _reload_store(self) -> None:
         """Reload store and check for addon updates."""
         await self.sys_store.reload()
+
+    @Job(name="tasks_reload_updater")
+    async def _reload_updater(self) -> None:
```

Comment:
Oh shoot good catch. I forgot to change that 😬

Reply:
Fixed. Though at some point I need to figure out how to make real tests for this that actually use the scheduler. Instead of directly testing the tasks like we do currently. Proper testing would've caught that